logdetective/server/database/models/__init__.py

```diff
@@ -0,0 +1,12 @@
+from logdetective.server.database.base import Base
+from logdetective.server.database.models.merge_requests import MergeRequests, Comments, Reactions
+from logdetective.server.database.models.metrics import AnalyzeRequestMetrics, EndpointType
+
+__all__ = [
```

Comment:
@jpodivin I know you don't like these imports but otherwise we should import every new database model class explicitly [here](https://github.com/fedora-copr/logdetective/blob/0709391db1ee9bda84ecfcd6d4cd64861af8dd43/alembic/env.py#L8) every time we create a new one (to use the alembic autogenerate feature). 
Or we should go back as it was before, having only one `models.py` file, but it would be already more than 1000 lines long.

Reply:
I don't like to use alias if not strictly needed. I would keep the code as it is if you don't prefer to have just a single `models.py` file.